app/src/androidTest/java/com/epfl/drawyourpath/community/TournamentCreationFragmentTest.kt

```diff
@@ -176,9 +156,9 @@ class TournamentCreationFragmentTest {
         scenario.close()
     }
 
-    @Test

```

Comment:
Why this test is commented, maybe correct it or remove it if it not used.

Reply:
I will remove them